fhir-parent/pom.xml

```diff
@@ -29,11 +29,16 @@
     </licenses>
     <developers>
         <developer>
-            <name>IBM Watson Health Development</name>
-            <email>fhirci@us.ibm.com</email>
-            <url>https://www.ibm.com/</url>
-            <organization>IBM Watson Health</organization>
-            <organizationUrl>http://www.ibm.com</organizationUrl>
+            <name>Lee Surprenant</name>
+            <email>lmsurpre@merative.com</email>
+            <organization>Merative</organization>
+            <organizationUrl>https://merative.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Robin Arnold</name>
+            <email>robin.arnold@merative.com</email>
```

Comment:
rarnold@merative.com

Reply:
spoke with robin and he decided to stick with the robin.arnold one